functions.php

```diff
@@ -72,6 +72,13 @@ function twentysixteen_setup() {
 	add_theme_support( 'post-thumbnails' );
 	set_post_thumbnail_size( 1200, 0, true );
 
+	/*
+	 * Add image size to enhance responsive image functionality
+	 *
+	 * @link https://developer.wordpress.org/reference/functions/add_image_size/
+	 */
+	add_image_size( 'intermediate-width', 600, 600 );
```

Comment:
If the image size handle is not prefixed then there is a chance that a plugin could define the same name. The two possibilities would be to name it `twentysixteen-intermediate-width` or `600x600`. Further reading: https://github.com/grappler/wp-standard-handles#image-sizes